src/librustc_resolve/lib.rs

```diff
@@ -68,7 +72,8 @@ use syntax::ast::{ExprLoop, ExprWhile, ExprMethodCall};
 use syntax::ast::{ExprPath, ExprStruct, FnDecl};
 use syntax::ast::{ForeignItemFn, ForeignItemStatic, Generics};
 use syntax::ast::{Ident, ImplItem, Item, ItemConst, ItemEnum, ItemExternCrate};
-use syntax::ast::{ItemFn, ItemForeignMod, ItemImpl, ItemMac, ItemMod, ItemStatic, ItemDefaultImpl};
+use syntax::ast::{ItemFn, ItemForeignMod, ItemImpl, ItemMac, ItemMod,
+                  ItemStatic, ItemDefaultImpl};
```

Comment:
I intend for it to do what @tbu- wants - grouping and ungrouping imports. But that is pretty tricky to do. One day...

@pnkfelix I'm keen for rustfmt to do some quite advanced things - even more so than this kind of thing, e.g., convert glob imports to list imports. I hope that leaving these things as configurable rather than mandatory will assuage those who believe rustfmt should only change white space.